### PR TITLE
Add get user query

### PR DESCRIPTION
**Pull request by sidiousvic**
# Summary
Adds user query together with test. Graphene schema updated as well.

Query takes an `id` parameter of the type `ID`.

```gql
    query($id: ID) {
        user(id: $id) {
            id
            email
            firstName
            lastName
            balance
          	currency
        }
    }
```

# Changes
- Add docs directory
- Move migrations directory
- Update fixtures copy
- Add get users query

### CODE REVIEW

File: itzhak/fixtures/users.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
     "pk": 1,
     "fields": {
       "password": "wv]`E]~gU^5W%B8",
-      "email": "douglas@itzhak.com",
+      "email": "douglas@itzhak.money",
       "first_name": "douglas",
-      "last_name": "sidious",
+      "last_name": "mbogo ntongai",
```
Comment (sidiousvic) [inline itzhak/fixtures/users.json:9]: Hey @salitosmbogz is this your correct last name?

File: itzhak/tests/test_users.py [hunk 2]
```diff
@@ -19,14 +19,39 @@
     }
 """
 
+user_query = """
+    query($id: ID) {
```
Comment (sidiousvic) [inline itzhak/tests/test_users.py:23]: @salitosmbogz do you think this is an ergonomic way to query a user? I was thinking maybe we want to query by `name`, not `id`, what do you think?

File: itzhak/io/schema.py [hunk 3]
```diff
@@ -11,9 +11,13 @@ class Meta:
 
 class Query(graphene.ObjectType):
     users = graphene.List(UserType)
+    user = graphene.Field(UserType, id=graphene.ID())
```
Comment (sidiousvic) [inline itzhak/io/schema.py:14]:
```suggestion
    user = graphene.Field(UserType, name=graphene.String())
```
@salitosmbogz ♻️ About the `name` vs. `id` comment.

File: itzhak/fixtures/users.json (hunk 1 above)
Comment (numdouglas) [reply itzhak/fixtures/users.json:9]: yes it is.

File: itzhak/tests/test_users.py (hunk 2 above)
Comment (numdouglas) [reply itzhak/tests/test_users.py:23]: I'd prefer ID to maintain the uniqueness of the user.

File: itzhak/io/schema.py (hunk 3 above)
Comment (numdouglas) [reply itzhak/io/schema.py:14]: ♻️

File: itzhak/tests/test_users.py [hunk 4]
```diff
@@ -19,14 +19,39 @@
...
     def setUp(self):
         self.client = Client(schema)
-        self.user = mixer.blend(User)
 
+    # get all users
     def test_users_query(self):
-        response = self.client.execute(users_query, variables={"id": self.user.id})
+        mixer.blend(User)
+        mixer.blend(User)
+        mixer.blend(User)
```
Comment (numdouglas) [inline itzhak/tests/test_users.py]: maybe use a for loop for this?

Comment (sidiousvic) [reply itzhak/tests/test_users.py]: Great idea @salitosmbogz ! Thank you. Reworked.

**Review by numdouglas — APPROVED** [review]
Great work @sidiousvic

**Pull request merged**